### PR TITLE
Fix more django 2.0 errors

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

File: rest_framework_extensions/serializers.py [hunk 1]
```diff
@@ -28,7 +28,10 @@ def save(self, **kwargs):
 
     def update(self, instance, validated_attrs):
         for attr, value in validated_attrs.items():
-            setattr(instance, attr, value)
+            if hasattr(getattr(instance, attr), 'set'):
```
Comment (dgilge) [inline rest_framework_extensions/serializers.py]:
```python
if attr != 'update_fields' and hasattr(getattr(instance, attr), 'set'):
```
or
```python
if hasattr(getattr(instance, attr, None), 'set'):
```
should fix https://travis-ci.org/chibisov/drf-extensions/jobs/350934079#L481

**Comment by auvipy** [comment]
thanks

File: rest_framework_extensions/serializers.py (hunk 1 above)
Comment (auvipy) [reply rest_framework_extensions/serializers.py]: yes only 1.11+ supported

**Comment by auvipy** [comment]
could you do another round of review? I will add allow failure to matrix

**Pull request merged**